_assets/ci/Jenkinsfile

```diff
@@ -0,0 +1,57 @@
+pipeline {
+  
+  /* This image comes from _assets/docker */
+  agent {
+    docker {
+      label 'linux'
+      image 'statusteam/clojure:latest'
+    }
+  }
+
+  options {
+    timestamps()
+    /* avoid race conditions on deploy */
+    disableConcurrentBuilds()
+    /* manage how many builds we keep */
+    buildDiscarder(logRotator(numToKeepStr: '60'))
+  }
+
+  environment {
+    GH_REPO = 'status-im/extensions-fiddle'
+    GH_USER = 'status-im-auto'
+    GH_MAIL = 'auto@status.im'
+    HOME    = '/tmp'
+  }
+
+  stages {
+    stage('Prep') {
+      steps {
+        sh "git config user.name  ${GH_USER}"
+        sh "git config user.email ${GH_MAIL}"
+        sh 'yarn install'
+      }
+    }
+
+    stage('Build') {
+      steps {
+        sh 'make compile'
+      }
+    }
+
+    stage('Publish') {
+      steps { script {
+        test = ''
```

Comment:
Ah damn, forgot to remove this line. @jeluard please drop it in one of your changes. Cheers.